lang/cs/Org.Apache.REEF.Tests/Functional/Bridge/TestFailedEvaluatorEventHandler.cs

```diff
@@ -65,8 +72,18 @@ public IConfiguration DriverConfigurations()
                 .Build();
         }
 
-        private sealed class FailedEvaluatorDriver : IObserver<IDriverStarted>, IObserver<IAllocatedEvaluator>, IObserver<ICompletedEvaluator>
+        private void ValidateSuccessForFailedEvaluator(string testFolder)
```

Comment:
What about generalizing this function to get one more string parameter, `FailedEvaluatorMessage`? I think we can use this logic for the other test suites to be implemented later. And also what about moving this function into some kind of *utility* package?